src/colors/color.rs

```diff
@@ -337,6 +337,48 @@ impl Color {
         (long, medium, short)
     }
 
+    /// convert normalized RGB value to linear value
```

Comment:
```suggestion
    /// Convert a normalized RGB value to a linear value
```
Please also add more information about this formula, e.g. a link to the source, like below.